static/docs/get-started/share-data.md

```diff
@@ -15,6 +15,10 @@ pulled by your team or you when you need it.
 Usually, you run it along with `git commit` and `git push` to save changed
 [DVC-files](/doc/user-guide/dvc-file-format) to Git.
 
+The `dvc push` command allows one to upload data to remote storage. It doesn't
+save any changes in the code or DVC-files. Those should be saved by using      
+`git commit` and `git push`.
+
 See `dvc push` for more details and options for this command.
```

Comment:
Sorry, I was a little distracted last week. I do like the paragraph you guys came up with, but I will remove the paragraph separation above in my active PR for this doc. Thanks